tests/dags/providers/provider_api_scripts/test_smithsonian.py

```diff
@@ -45,29 +33,18 @@ def test_get_hash_prefixes_with_len_one():
         "e",
         "f",
     ]
-    actual_prefix_list = list(si._get_hash_prefixes(1))
+    actual_prefix_list = list(ingester._get_hash_prefixes())
     assert actual_prefix_list == expect_prefix_list
+    ingester.hash_prefix_length = 2  # Undo the change
```

Comment:
We'll need to put this in a `try`/`finally`, since if the assert fails here the reset on the class level value won't be performed:
```suggestion
    try:
        assert actual_prefix_list == expect_prefix_list
    finally:
        ingester.hash_prefix_length = 2  # Undo the change
```

Alternatively, you could set up an `ingester` fixture (here's an example from the Phylopic refactor):

https://github.com/WordPress/openverse-catalog/blob/6380223a52ab69428278ad48d9fe241d7a471e39/tests/dags/providers/provider_api_scripts/test_phylopic.py#L25-L31

Reply:
It's funny because this was a really lazy way to write this test 😂 Thanks for spotting it!
I tend to use the patch context manager, but it's great to know there are so many other ways 😮